tests/test_workspaces/test_helpers.py

```
@@ -0,0 +1,21 @@
+
+    workspace_general_setting = WorkspaceGeneralSettings.objects.filter(workspace_id=workspace_id).first()
+    workspace_general_setting.is_multi_currency_allowed = False
+    workspace_general_setting.save()
+
+    Workspace.objects.filter(id=workspace_id).update(fyle_currency='YOLO')
+    QBOCredential.objects.filter(workspace_id=workspace_id).update(currency='USD')
+
+    enable_multi_currency_support(workspace_general_settings=workspace_general_setting)
+
+    workspace_general_setting = WorkspaceGeneralSettings.objects.filter(workspace_id=workspace_id).first()
+    assert workspace_general_setting.is_multi_currency_allowed == True
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if ...:` for truth checks

Replace comparison

(E712)